bin/test/main.ml

```diff
@@ -393,6 +393,7 @@ let report_error_in_block block msg =
 let run setup non_deterministic not_verbose syntax silent verbose_findlib
     prelude prelude_str file section root direction force_output output : int =
     try
+      Printexc.record_backtrace false;
```

Comment:
It looks like we do still need it, sadly -- at least the test/bin/mdx-test/expect/prelude-file/test-case.md test fails, printing this extra bit:

> Called from file "toplevel/toploop.ml", line 208, characters 17-27

Reply:
Ok, in that case I think it would be nice to disable backtraces by default but to expose a cli flag so that it can be controlled by users if they want to.

What do you think?

Reply:
Sounds good, I will add the flag!